### DISCUSSION TITLE
bagisto sql error

### DISCUSSION THREAD

**Discussion by vishaldamasiya** [bagisto/bagisto#5844] [Q&A]
SQLSTATE[HY000]: General error: 1267 Illegal mix of collations (utf8mb4_unicode_ci,IMPLICIT) and (utf8mb4_general_ci,IMPLICIT) for operation '=' (SQL: update `categories` set `_lft` = case when `_lft` between 15 and 16 then `_lft`-1 when `_lft` between 14 and 16 then `_lft`+2 else `_lft` end, `_rgt` = case when `_rgt` between 15 and 16 then `_rgt`-1 when `_rgt` between 14 and 16 then `_rgt`+2 else `_rgt` end where (`_lft` between 14 and 16 or `_rgt` between 14 and 16))

**Comment by Sanjaybhattwebkul** [comment]
Hi @vishaldamasiya 
Please create a databse with u**tf8mb4_unicode_ci** collection.
Then the issue will resolve.

**Comment by vishaldamasiya** [comment]
SQLSTATE[HY000]: General error: 1267 Illegal mix of collations (utf8mb4_unicode_ci,IMPLICIT) and (utf8mb4_general_ci,IMPLICIT) for operation '=' (SQL: update `categories` set `image` = category/2/wIWq51IUaraHGk7da3uDVKZE0MyuEn4mG7HlxpUH.jpg, `categories`.`updated_at` = 2022-02-17 15:11:15 where `id` = 2)

**Comment by Sanjaybhattwebkul** [comment]
@vishaldamasiya 
sorry, the database collection will be **utf8_unicode_ci**

**Comment by vishaldamasiya** [comment]
SQLSTATE[HY000]: General error: 1267 Illegal mix of collations (utf8mb4_unicode_ci,IMPLICIT) and (utf8mb4_general_ci,IMPLICIT) for operation '=' (SQL: update `categories` set `_lft` = case when `_lft` >= 14 then `_lft`+2 else `_lft` end, `_rgt` = case when `_rgt` >= 14 then `_rgt`+2 else `_rgt` end where (`_lft` >= 14 or `_rgt` >= 14))

**Comment by Sanjaybhattwebkul** [comment]
Please recreate the database with utf8_unicode_ci collation.

**Comment by vishaldamasiya** [comment]
[
![image](https://user-images.githubusercontent.com/99727207/154450021-1f921f58-e81c-419c-bc7f-8ccf0c38a40a.png)
](url)

**Comment by Sanjaybhattwebkul** [comment]
@vishaldamasiya 
I hope your issue will be resolved. if still, you are getting the same issue. then please let me know.
